client/my-sites/activity/filterbar/text-selector.tsx

```diff
@@ -37,7 +30,7 @@ const TextSelector: FunctionComponent< Props > = ( { siteId, filter } ) => {
 			initialValue={ filter.textSearch || null }
 			isOpen={ false }
 			onSearch={ handleSearchActivities }
-			placeholder={ placeholder }
+			placeholder={ translate( 'Search posts by ID, title or author' ) }
```

Comment:
Discussing with @keoshi, given we have now a full width text search box in mobile, we can use the full placeholder instead of just `Search posts`

Reply:
Nice win here, @Initsogar !

Could we adjust it to add the Oxford comma at the end and have it read `Search posts by ID, title, or author` ?

Reply:
Sure. I will proceed in a new PR, so I can wait for translation before merging it.